server/src/main/java/com/higgs/server/web/svc/AuthenticationService.java

```diff
@@ -71,6 +71,6 @@ public UserLogin performUserSearch(@NonNull final String name) {
         if (userLogin.isPresent()) {
             return userLogin.get();
         }
-        throw new UsernameNotFoundException(name);
+        throw new BadCredentialsException(String.format("Failed to authenticate user %s", name));

```

Comment:
make sure this and other exception messages like it get logged on the server logs. if it doesn't, add it.